frontend/src/components/utilities/parseDotEnv.ts

```diff
@@ -6,7 +6,7 @@ const LINE =
  * @param {ArrayBuffer} src - source buffer
  * @returns {String} text - text of buffer
  */
-export function parseDotEnv(src: ArrayBuffer) {
```

Comment:
I would move the parseJson function into the same file as the parseDotEnv function, since their use-case is very much alike. 

I suggest creating a single filed called `parseEnv.ts`.